docs/UserGuide.adoc

```diff
@@ -98,6 +99,39 @@ Only valid if a web-page is currently being viewed.
 * `*add* u/nus-cs2103-ay1920s1.github.io n/Module Website f/CS2103T`
 * `*add* u/www.youtube.com/watch?v=9AMcN-wkspU n/IntelliJ Tips and Tricks t/video t/watch-later`
 
+=== Adding a new folder *`folder`*
+
+Creates a new folder for organizing bookmarks into.
+
+Format: `*newfolder* FOLDER_NAME [p/PARENT_FOLDER]`
+
+****
+* `FOLDER_NAME` must not already exist.
+* `PARENT_FOLDER` must be an existing folder.
+* If `PARENT_FOLDER` is not specified, `FOLDER_NAME` will have no parent, and be at the root.
```

Comment:
```suggestion
* If `PARENT_FOLDER` is not specified, `FOLDER_NAME` will have no parent and will be placed at the root.
```

Reply:
This I'm not sure which would be better, but personally I feel it's easier to read with less comma splicing

Reply:
ok!